GPUQREngine/Source/BucketList/BucketList_CreateBundles.cpp

```diff
@@ -13,12 +13,11 @@
 //
 // =============================================================================
 
-#include "GPUQREngine_BucketList.hpp"
-
-
 // CreateBundles selects rowtiles up to PANELSIZE and creates a new bundle
 // ready for factorization.
-void BucketList::CreateBundles
+#include "GPUQREngine_BucketList.hpp"
```

Comment:
IIUC, the comment above this line belongs to the function below. Could please move the `#include` statement above again?